examples/vmod_error/tests/test01.60.vtc

```diff
@@ -1,6 +1,6 @@
 varnishtest "error()"
 
-feature cmd { varnishd -V 2>&1 | head -n 1 | grep varnish-6.0. }
+feature cmd { varnishd -V 2>&1 | head -n 1 | grep -E varnish(-plus)?-6.0. }
```

Comment:
if this is a regex, shouldn't it be inside quotes, with the `.` escaped?

Reply:
suuuuuuuuuuuure